test/e2e/network/ingress.go

```diff
@@ -935,3 +940,190 @@ func detectNegAnnotation(f *framework.Framework, jig *e2eingress.TestJig, gceCon
 		framework.ExpectNoError(err)
 	}
 }
+
+var _ = SIGDescribe("Ingress API", func() {
+	f := framework.NewDefaultFramework("ingress")
+	/*
+		Release: v1.19
+		Testname: Ingress API
+		Description:
+		- The networking.k8s.io API group MUST exist in the /apis discovery document.
+		- The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
+		- The ingresses, ingressclasses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
+		- The networking resource must support create, get, list, watch, update, patch, delete, and deletecollection.
+		- The networking/ingresses resource must support get, update, patch.
+		- The networking/ingressclasses resource must support get, update, patch.
+	*/
+
+	ginkgo.It("should support creating Ingress API operations", func() {
+		// Setup
+		ns := f.Namespace.Name
+
+		// this test wants powerful permissions.  Since the namespace names are unique, we can leave this
+		// lying around so we don't have to race any caches
+		err := e2eauth.BindClusterRole(f.ClientSet.RbacV1(), "cluster-admin", f.Namespace.Name,
+			rbacv1.Subject{Kind: rbacv1.ServiceAccountKind, Namespace: f.Namespace.Name, Name: "default"})
+		framework.ExpectNoError(err)
+
+		err = e2eauth.WaitForAuthorizationUpdate(f.ClientSet.AuthorizationV1(),
+			serviceaccount.MakeUsername(f.Namespace.Name, "default"),
+			"", "create", schema.GroupResource{Resource: "pods"}, true)
+		framework.ExpectNoError(err)
```

Comment:
I don't think these lines are required... is the copied from somewhere else?

Reply:
Yes think this was a copy/paste error